components/About/Profile.js

```diff
@@ -29,15 +12,15 @@ const LinkedIn = styled(FaLinkedin)`
 
 export default function Profile(props) {
   return (
-    <Aside>
-      <Image1 src={props.image} alt='Eddy' width={200} height={200} />
+    <div class="text-left max-w-[65ch] ">
+      <Image class="my-0 mx-auto rounded-[50%] block" src={props.image} alt='Eddy' width={200} height={200} />
```

Comment:
@drebarrera there's a few spots where you used class, replace those with classname

Reply:
All fixed!